controller/internal/enforcer/nfqdatapath/nfq_linux.go

```diff
@@ -107,6 +136,26 @@ func (d *Datapath) processNetworkPacketsFromNFQ(p *nfqueue.NFPacket) {
 		p.QueueHandle.SetVerdict2(uint32(p.QueueHandle.QueueNum), 1, uint32(p.Mark), uint32(copyIndex), uint32(p.ID), buffer)
 
 	}
+	if netPacket.IPProto == packet.IPProtocolTCP {
+		d.collectTCPPacket(&debugpacketmessage{
+			Mark:    p.Mark,
+			p:       netPacket,
+			tcpConn: tcpConn,
+			udpConn: nil,
+			err:     nil,
+			network: true,
+		})
+	} else if netPacket.IPProto == packet.IPProtocolUDP {
+		d.collectUDPPacket(&debugpacketmessage{
+			Mark:    p.Mark,
+			p:       netPacket,
+			tcpConn: nil,
+			udpConn: udpConn,
+			err:     nil,
+			network: true,
+		})
+	}
+
```

Comment:
Same as above here. We cannot report all dropped packets. The overhead is too high.